File: python3/1957.py
class Solution:
    def makeFancyString(self, s: str) -> str:

        res = ""
        c1, c2 = None, None
        for c in s:
            if c != c1 or c != c2:
                res += c
                c1,c2 = c, c1
        return res
        

File: rust/1957.rs
impl Solution {
    pub fn make_fancy_string(s: String) -> String {
        let mut res = String::new();
        let mut c1: Option<char> = None;
        let mut c2: Option<char> = None;

        for c in s.chars() {
            if Some(c) != c1 || Some(c) != c2 {
                res.push(c);
                c2 = c1;
                c1 = Some(c);
            }
        }
        res
    }
}
